src/App.js

```diff
@@ -71,26 +67,45 @@ const App = ({ locationId }) => {
       });
 
       const responseLocations = response.result.values
-        .map(location => ({
-          id: location[0],
-          name: location[1],
-          address: location[2],
-          latitude: location[3] ? parseFloat(location[3]) : 0,
-          longitude: location[4] ? parseFloat(location[4]) : 0,
-          category: location[5],
+        .map(([
+          id,
+          name,
+          address,
+          latitude,
+          longitude,
+          category,
+          nursing,
+          stroller,
+          changeTable,
+          indoor,
+          outdoor,
+          description,
+          strollerTips,
+          nursingTips,
+          website,
+          instagram,
+          facebook,
```

Comment:
This is a small cleanup I would have made separate. I agree this is nicer, but also an accidental mistake here would have the potential to totally bust the data of the app if the order is wrong. Although I do really like this change! This will help avoid errors if we ever add a new column in the middle (vs. needing to update all the indices on all the lines... literally a bug happened previously when I made a typo doing that once).

Reply:
Yeah that makes sense. I ended up doing it in this PR because I needed to follow the flow of the lat/lon props.